src/OpenSage.Game/Logic/Object/Locomotor.cs

```diff
@@ -1025,9 +1025,9 @@ private void MoveTowardsPositionWheels(GameObject obj, PhysicsBehavior physics,
             effectiveSlowDownDist = 1 * AIPathfind.PathfindCellSizeF;
         }
 
-        const float FIFTEEN_DEGREES = MathF.PI / 12.0f;
+        const float fifteenDegrees = MathF.PI / 12.0f;
         var projectFrames = _gameEngine.LogicFramesPerSecond / 2; // Project out 1/2 second.
-        if (MathF.Abs(relAngle) > FIFTEEN_DEGREES)
+        if (MathF.Abs(relAngle) > fifteenDegrees)
         {
             // If we're turning more than 10 degrees, check & see if we're moving into "impassable territory"
```

Comment:
Assuming this is just ported over (and I realize you're just renaming this), but interesting that this is named fifteen degrees, but the comment says 10 degrees.

Reply:
Ha, well spotted. I've updated the comment.